packages/host/app/components/operator-mode/container.gts

```diff
@@ -791,6 +793,7 @@ export default class OperatorModeContainer extends Component<Signature> {
               {{svgJar 'download' width='30px' height='30px'}}
             </button>
           {{/if}}
+          <SubmodeSwitcher @submode={{this.subMode}} @onSubmodeSelect={{fn (mut this.subMode)}} />
```

Comment:
I know I'm getting insanely annoying with my OCD tendencies but for consistency decide whether you wanna use `submode` or `subMode`